core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -653,6 +653,13 @@ private void updateAndPushParents(WorkflowModel workflow, String operation) {
             // update parent's sub workflow task
             TaskModel subWorkflowTask =
                     executionDAOFacade.getTaskModel(workflow.getParentWorkflowTaskId());
+            if (subWorkflowTask.getWorkflowTask() != null
```

Comment:
The null check is redundant, In a TaskModel instance, `WorkflowTask` should always be present.